consul/datadog_checks/consul/consul.py

```diff
@@ -77,18 +77,9 @@ def __init__(self, name, init_config, instances):
         self._instance_states = defaultdict(lambda: ConsulCheckInstanceState())
 
         self.HTTP_CONFIG_REMAPPER = {
-            'client_cert_file': {
-                'name': 'tls_cert',
-                'default': self.init_config.get('tls_cert') or self.init_config.get('client_cert_file') or False,
-            },
-            'private_key_file': {
-                'name': 'tls_private_key',
-                'default': self.init_config.get('tls_private_key') or self.init_config.get('private_key_file') or False,
-            },
-            'ca_bundle_file': {
```

Comment:
`client_cert_file` is used to set the value of `verify` only if its a string:
```
        if isinstance(config['tls_ca_cert'], string_types):
            verify = config['tls_ca_cert']
        elif not is_affirmative(config['tls_verify']):
            verify = False
```
So the default to False has no effect. It was also not documented as init config